scripts/linux/runE2ETest.sh

```diff
@@ -1085,6 +1073,12 @@ TRANSPORT_TYPE_4="${TRANSPORT_TYPE_4:-mqtt}"
 TWIN_UPDATE_FAILURE_THRESHOLD="${TWIN_UPDATE_FAILURE_THRESHOLD:-00:01:00}"
 METRICS_SCRAPE_FREQUENCY_IN_SECS="${METRICS_SCRAPE_FREQUENCY_IN_SECS:-300}"
 METRICS_UPLOAD_TARGET="${METRICS_UPLOAD_TARGET:-AzureLogAnalytics}"
+
+if [[ "${TEST_NAME,,}" == "longhaul" ]] ||
+   [[ "${TEST_NAME,,}" == "stress"  ]]; then
+    tracking_id=$(cat /proc/sys/kernel/random/uuid)
```

Comment:
The rule we agreed is that TestInfo should be updated in yaml only except device id.  For tracking id, your call, either leave here or move to yaml.

Reply:
I'll open a separate PR